matrix_messages.rst

```diff
@@ -0,0 +1,92 @@
+Raiden transport messages
+#########################
+
+Definitions
```

Comment:
These should not be here. We have a terminology document. Please simply use terms from there or add new ones there if they don't already exist.

Reply:
Noted; I'll move it there once we settle on format.